tests/spdx/examples/test_spdx2_document_from_scratch.py

```diff
@@ -0,0 +1,143 @@
+            Checksum(ChecksumAlgorithm.MD5, "624c1abb3664f4b35547e7c73864ad24"),
+        ],
+        license_concluded=get_spdx_licensing().parse("GPL-2.0-only OR MIT"),
+        license_info_from_files=[get_spdx_licensing().parse("GPL-2.0-only"), get_spdx_licensing().parse("MIT")],
+        license_declared=get_spdx_licensing().parse("GPL-2.0-only AND MIT"),
+        license_comment="license comment",
+        copyright_text="Copyright 2022 Jane Doe",
+        description="package description",
+        attribution_texts=["package attribution"],
+        primary_package_purpose=PackagePurpose.LIBRARY,
+        release_date=datetime(2015, 1, 1),
+        external_references=[
+            ExternalPackageRef(
+                category=ExternalPackageRefCategory.OTHER,
+                reference_type="http://reference.type",
+                locator="reference/locator",
+                comment="external reference comment",
+            )
+        ],
+    )
+
+    # Now that we have a package defined, we can add it to the document's package property.
+    document.packages = [package]
+
+    # A DESCRIBES relationship asserts that the document indeed describes the package.
+    describes_relationship = Relationship("SPDXRef-DOCUMENT", RelationshipType.DESCRIBES, "SPDXRef-Package")
+    document.relationships = [describes_relationship]
+
+    # Let's add two files. Have a look at the file class for all possible properties a file can have.
+    file1 = File(
+        name="./package/file1.py",
+        spdx_id="SPDXRef-File1",
+        file_types=[FileType.SOURCE],
+        checksums=[
+            Checksum(ChecksumAlgorithm.SHA1, "d6a770ba38583ed4bb4525bd96e50461655d2758"),
+            Checksum(ChecksumAlgorithm.MD5, "624c1abb3664f4b35547e7c73864ad24"),
+        ],
+        license_concluded=get_spdx_licensing().parse("MIT"),
+        license_info_in_file=[get_spdx_licensing().parse("MIT")],
+        copyright_text="Copyright 2022 Jane Doe",
+    )
+    file2 = File(
+        name="./package/file2.py",
+        spdx_id="SPDXRef-File2",
+        checksums=[
+            Checksum(ChecksumAlgorithm.SHA1, "d6a770ba38583ed4bb4525bd96e50461655d2759"),
+        ],
+        license_concluded=get_spdx_licensing().parse("GPL-2.0-only"),
+    )
+
+    # Assuming the package contains those two files, we create two CONTAINS relationships.
+    contains_relationship1 = Relationship("SPDXRef-Package", RelationshipType.CONTAINS, "SPDXRef-File1")
+    contains_relationship2 = Relationship("SPDXRef-Package", RelationshipType.CONTAINS, "SPDXRef-File2")
+
+    # This library uses run-time type checks when assigning properties.
+    # Because in-place alterations like .append() circumvent these checks, we don't use them here.
+    document.relationships += [contains_relationship1, contains_relationship2]
+    document.files += [file1, file2]
+
+    # We now have created a document with basic creation information, describing a package that contains two files.
```

Comment:
there should also be assertions, that the right number of files and relationships are in the document, to check for consistency.

Reply:
I added the assertions